.github/workflows/release-published.yml

```diff
@@ -539,32 +539,12 @@ jobs:
           
           done
             
-  publish_assets_to_internal_nexus:
+  publish_assets_to_s3_bucket:
     if: ${{ inputs.dry_run == false }}
-    name: Publish OSS released assets to repo.liquibase.net
+    name: Publish OSS released assets to s3 bucket liquibaseorg-origin
     needs: [ setup ]
     runs-on: ubuntu-22.04
-    env:
-      MAVEN_USERNAME: ${{ secrets.INTERNAL_NEXUS_USERNAME }}
-      MAVEN_PASSWORD: ${{ secrets.INTERNAL_NEXUS_PASSWORD }}
-      GPG_PASSWORD: ${{ secrets.GPG_PASSPHRASE }}
     steps:
-      # Check in nexus if the folder with the version already exists, if exists exit with error code
-      - name: Check if the version already exists in nexus
```

Comment:
Aren't we missing this check now? Cause in a rerun we would overwrite them in S3.
Or we could add a timestamp to S3 directory and so even if we rerun it we'll have a new directory in S3 that we can remove later on.